.github/workflows/ci.yaml

```diff
@@ -0,0 +1,54 @@
+name: CI
+on: [push]
+
+jobs:
+  lint:
+    name: Lint
+
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        module:
+          - slog
+
+    steps:
+
+    - name: Set up Go ${{ matrix.go }}
+      uses: actions/setup-go@v1
+      with:
+        go-version: "1.20"
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: lint
+      uses: golangci/golangci-lint-action@v3
+      with:
+        version: "v1.52.2"
+        working-directory: ${{matrix.module}}
+
+  test:
+    name: Test
+
+    runs-on: ubuntu-22.04
+
+    strategy:
+      matrix:
+        go:
+          - "1.20"
```

Comment:
since we may use different Go versions around different services at least in a time window, we can test with different Go versions